cmd/skupper/README.md

```diff
@@ -0,0 +1,41 @@
+# Skupper CLI
+
+The [Skupper](https://skupper.io/) command line enables you to:
+
+* Create a Virtual Application Network (VAN) site
+* Connect to other VAN sites
+* Connect services
+
+## Getting `skupper`
+
+To install `skupper`, download the [latest release](https://github.com/skupperproject/skupper/releases).
+
+
+## Using `skupper`
+
+To create your first VAN site:
+
+```
+skupper init
```

Comment:
My inclination would be to separate `skupper init` from the creation of the connection token. E.g.

To initialise a namespace to use skupper, run `skupper init` against it. This can be undone by `skupper delete`.

To connect one namespace using skupper to another, you first need a connection token from the target namespace which you can obtain using `skupper connection-token /apth/to/mysecret.yaml`. The token written into the path specified can then be used at the other namespace with `skupper connect /path/to/mysecret.yaml`. After some time you can verify these are connected using `skupper status`.

Reply:
@grs  is there a difference between 'site' and 'namespace'? ie could I take your suggestion but use 'site'?

Reply:
Yes, I think that would be fine.